src/routes/components/components.json

```
@@ -2676,5 +2676,14 @@
 		"category": "User Interaction",
 		"tags": ["animations", "components and libraries", "interactions"],
 		"stars": 0
-	}
+	},
+  	{
+    		"title": "svelte-switch-case",
+		"url": "https://github.com/l-portet/svelte-switch-case",
+		"description": "Switch case syntax for Svelte",
+		"npm": "svelte-switch-case",
+		"addedOn": "2022-07-25",
+		"tags": ["preprocessors"],
+    		"stars": 47
```

There are spaces instead of a tabulation
```suggestion
		"stars": 47
```